roaster-restassured/src/main/java/hu/icellmobilsoft/roaster/restassured/response/ResponseProcessor.java

```diff
@@ -0,0 +1,191 @@
+package hu.icellmobilsoft.roaster.restassured.response;
+
+import hu.icellmobilsoft.roaster.restassured.annotation.JSON;
+import hu.icellmobilsoft.roaster.restassured.annotation.XML;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import io.restassured.specification.ResponseSpecification;
+import org.eclipse.microprofile.config.ConfigProvider;
+
+import javax.enterprise.inject.spi.CDI;
+
+/**
+ * Base Response REST handler
+ *
+ * @param <RESPONSE> response class (any type)
+ * @author imre.scheffer
+ * @since 0.2.0
+ */
+public abstract class ResponseProcessor<RESPONSE> {
+
+    /**
+     * Base URI config key
+     * 
+     * @return config key like "project.service.base.uri"
+     */
+    public abstract String baseUriKey();
+
+    /**
+     * Get value by {@link #baseUriKey()} from microprofile config
+     * 
+     * @return value like "http://localhost:8080"
+     */
+    public String baseUri() {
+        return ConfigProvider.getConfig().getValue(baseUriKey(), String.class);
+    }
+
+    /**
+     * HTTP path to call
+     * 
+     * @return value like "/test/service/generate/testData"
+     */
+    public abstract String path();
+
+    /**
+     * Call and get JSON object from HTTP GET method
+     * 
+     * @param responseClass
+     *            response class
+     * @return response object casted to responseClass
+     */
+    public RESPONSE getJson(Class<RESPONSE> responseClass) {
```

Comment:
Utolsó paraméterként szerintem fel lehetne venni a path paramétereket is, pl. Object... pathParams formában, hogy ne a path override-nál kelljen ezzel bajlódni (pl. "/test/service/generate/{id}" jellegű path-oknál)